packages/atomic/src/components/common/interface/bindings.ts

```diff
@@ -1,13 +1,34 @@
 import {HTMLStencilElement} from '@stencil/core/internal';
 import {i18n} from 'i18next';
 import {AnyEngineType} from './interface-common';
+import {AtomicCommonStoreData} from './store';
+
+export interface ExtensibleStencilStore<
+  StoreData extends AtomicCommonStoreData
```

Comment:
It would remove the need to explicitly set the type, yes.

However, it's a bit similar to having default argument values for function: They should generally be used when you want to convey to the users of the interface that they should probably not pass in a value for a given function argument the majority of the time in order to "use the function properly".

Here I feel it's better to always require the user of the interface to explicitly specify the type of data the store hold in order to "use the interface correctly", versus having the common one be the default.

So it's more in the spirit of instructing users how the interface should normally be used.

Does it make sense ?